### PR TITLE
DOC-2300: rename `Enhanced Image Editing` with `Image Editing` for TinyMCE 7.0

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2300

Site: [DOC-2300_DOC-2309 site](http://docs-feature-70-doc-2300doc-2309.staging.tiny.cloud/docs/tinymce/latest/editimage/)

Changes:
* Rename `Enhanced Image Editing` with `Image Editing` for TinyMCE 7.0

Pre-checks:
- [x] Branch prefixed with `feature/7/` or `hotfix/7/`
- [x] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [x] Files has been included where required `(if applicable)`
- [x] Files removed have been deleted, not just excluded from the build `(if applicable)`
- [x] Files added for `New product features`, and included a `release note` entry.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by TheSpyder — APPROVED** [review]

File: modules/ROOT/pages/editor-command-identifiers.adoc [hunk 1]
```diff
@@ -329,10 +329,10 @@ The following commands require the xref:directionality.adoc[Directionality (`+di
 include::partial$commands/directionality-cmds.adoc[leveloffset=+3]
 
 
-[[editimage]]
-==== Enhanced Image Editing
+[[imageediting]]
```
Comment (MitchC1999) [inline modules/ROOT/pages/editor-command-identifiers.adoc:332]:
```suggestion
[[editimage]]
```

File: modules/ROOT/pages/troubleshoot-server.adoc [hunk 2]
```diff
@@ -14,6 +14,7 @@ If a service does not appear to be working, this is generally caused by one of t
 
 . The `+application.conf+` file is incorrect. Please go back and follow the steps listed in the xref:introduction-to-premium-selfhosted-services.adoc#create-a-configuration-file[installation guide]. This is the most common problem - often the origins are specified without the port numbers and this can cause things to fail, eg: use `+http://localhost:8080+` instead of `+http://localhost+`. After making changes to the `+application.conf+` file, please restart your Java web server (e.g. Jetty or Tomcat).
 . The `+application.conf+` file is correct, but something is wrong with one of the services. See the section below to debug the services.
+. The `+application.conf+` file is correct, and the services are working, but the server-side component URLs that the editor uses are not quite right. Refer to the xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro], xref:editimage.adoc[Image Editing], xref:introduction-to-mediaembed.adoc[Enhanced Media Embed] and xref:linkchecker.adoc[Link Checker] plugin pages for help.
```
Comment (MitchC1999) [inline modules/ROOT/pages/troubleshoot-server.adoc:17]: Looks like this is duplicated

File: modules/ROOT/partials/module-loading/bundling-plugin-files.adoc [hunk 3]
```diff
@@ -36,6 +35,7 @@ include::partial$module-loading/bundling-plugins-that-cant-bundle.adoc[]
 ** xref:full-screen[Full Screen (fullscreen)]
 ** xref:help[Help (help)]
 ** xref:image[Image (image)]
+** xref:enhanced-image-editing[Image Editing (editimage)]
```
Comment (MitchC1999) [inline modules/ROOT/partials/module-loading/bundling-plugin-files.adoc:38]:
```suggestion
** xref:image-editing[Image Editing (editimage)]
```

**Review by ltrouton — APPROVED** [review]

File: modules/ROOT/pages/troubleshoot-server.adoc (hunk 2 above)
Comment (kemister85) [reply modules/ROOT/pages/troubleshoot-server.adoc:17]:
```suggestion
```

**Pull request merged**